src/sql/media/icons/common-icons.css

```diff
@@ -524,6 +524,15 @@ Includes non-masked style declarations. */
 	background-image: url('start-outline.svg');
 }
 
+.codicon:not(.masked-icon).icon-split-cell {
+	background-image: url("i_split_cell.svg");
```

Comment:
Why the i? I think just having the name `split_cell.svg` is enough.

(although personally I also think a more generic name should be used. @halerankin do you know if there's a more "common" name we could give this icon?)

Reply:
@halerankin  - Can you pitch in here ?

Reply:
Renamed